src/lib/helpers/CameraHelper.svelte

```diff
@@ -0,0 +1,20 @@
+<script lang="ts">
+    import { CameraHelper, Camera } from 'three'
+    import Object3DInstance from '../instances/Object3DInstance.svelte'
+    import { getParent } from '../internal/HierarchicalObject.svelte'
+  
+    const parent = getParent()
+  
+    let helper: CameraHelper
+  
+    if (!($parent instanceof Camera)) {

```

Comment:
This helper poses the same problem as the BoxHelper, slightly more complex due to the object needing to be of type `Camera`. I propose:

```ts
const parent = getParent()
if (!($parent instanceof Camera)) throw new Error('…') // throwing is fine in this context as it's also a debugging utility.
export const cameraHelper = new CameraHelper($parent) // rename import, see ArrowHelper comment
export const update = () => cameraHelper.update()
```